contracts/vortex/CarbonVortex.sol

```diff
@@ -81,7 +125,22 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
      * @dev performs contract-specific initialization
      */
     function __CarbonVortex_init_unchained() internal onlyInitializing {
-        _setRewardsPPM(100_000);
+        // set rewards ppm to 5000
+        _setRewardsPPM(5000);
+        // set price reset multiplier to 2x
+        _setPriceResetMultiplier(2);
```

Comment:
Isn't this 2 by definition (of the product)?
If yes, then we can just use a (non-user-configurable) constant.

Reply:
it should be configurable